react/src/lib/components/DeckGLMap/DeckGLMap.jsx

```diff
@@ -1,6 +1,10 @@
 import PropTypes from "prop-types";
 import * as React from "react";
 import Map from "./components/Map";
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
I think it should be possible to fix this by using import statement

Reply:
Cannot use import statement in jsx file